src/misc/support.cpp

```diff
@@ -392,3 +392,25 @@ const std_fs::path &GetExecutablePath()
 	}
 	return exe_path;
 }
+
+const Bit8u DOS_DATE_months[] = {0,  31, 28, 31, 30, 31, 30,
```

Comment:
`constexpr uint8_t`

(In general, try to use `constexpr` whenever working with literals; it moves the evaluation to compile-time instead of at runtime.)

Reply:
Moved to support.h with constexpr. Done.